Makefile

```diff
@@ -1,4 +1,4 @@
-VERSION = 0.0.3
+VERSION = 0.0.4
```

Comment:
I was thinking about this version/tag when working on the last PR. Are we going to continue using this version/tag in github ? As we might have some overlap with our existing versions for cyclops in the future which might be confusing. Suggest we choose a different tag pattern, like `vX.Y.Z' here and might also want to bump this version a bit more since it has been running and tested internally for a while.

Reply:
@Jacobious52 and I agreed that once internal changes have been pushed upstream and we have deprecated the internal copy of the repo we would release a "1.0.0" version.

Changing the the version pattern is out of the scope for this PR

Reply:
Sounds good. Just need to be careful when we release 1.0.0 as it duplicates old image tag.